tests/test_chaining.py

```
@@ -7,26 +7,29 @@
 from pytest_postgresql.executor_noop import NoopExecutor
 
 
-def load_schema(host: str, port: int, user: str, dbname: str, password: str | None) -> None:
+def load_schema(host: str, port: int, user: str, dbname: str, password: str | None, autocommit: bool) -> None:
```

Ruff (0.15.21)

[warning] Boolean-typed positional argument in function definition

(FBT001)